src/index.ts

```diff
@@ -1,4 +1,23 @@
-export type Networks = 'rinkeby' | 'mainnet';
+/*
+    The reason for duplicating types in both uppercase/lowercase is because in the safe-react
+    type for networks contains uppercase strings and with previous type it resulted in a type error.
+    The sdk converts network to lowercase, so passing an uppercase one is totally valid too.
+*/
+export type Networks =
+  | 'MAINNET'
+  | 'MORDEN'
+  | 'ROPSTEN'
+  | 'RINKEBY'
+  | 'GOERLI'
+  | 'KOVAN'
+  | 'UNKNOWN'
```

Comment:
Why do we need unknown?

Reply:
It's a valid value shown in the interface when we don't know what network the user is connected to

Reply:
So there's a possibility it could be returned